templates/app/app.txt

```diff
@@ -47,7 +47,7 @@ requireAll({
 requireAll({
   dirname: path.join(__dirname, 'middlewares'),
   recursive: true,
-  resolve: Module => new Module(app),
+  resolve: Module => Module(app),
```

Comment:
[Cleaning] Not related To your feature but I think we should get rid of the `requireAll` and `middlewares` folder. (Those are relics of Lumber v2)

Reply:
What is you preferred?
1-Move files in middlewares folder (forestadmin.js & welcome.js) beside `app.js`
2-Move content of these files inside `app.js`
1, isn't it?